scrapy/settings/default_settings.py

```diff
@@ -230,9 +230,12 @@
 STATS_DUMP = True
 
 STATSMAILER_RCPTS = []
-
-TEMPLATES_DIR = abspath(join(dirname(__file__), '..', 'templates'))
-
+TEMPLATES_DIR_BASE = [
```

Comment:
I was suggesting that when we create the project, we add this setting to the scrapy.cfg (using startproject.py). So when we are running this command from inside the module (for genspider), you will automatically get the folder location as a part of `template_dir_base` right? I guess I am a little confused by what you mean by module.